config/database.yml

```diff
@@ -2,10 +2,10 @@ development:
   adapter: postgresql
```

Comment:
As I told you yesterday. Do not push database.yaml all the time. If any changes are there, just make it in your local but don't push